app/views/users/sessions/new.html.haml

```diff
@@ -3,29 +3,29 @@
 .auth-form.sign-in-form
 
   = form_for User.new, url: user_session_path, html: { class: "form" } do |f|
-    %h1.huge-title Connectez-vous
+    %h1.huge-title= t('views.sessions.new.title')
 
     = render partial: 'shared/france_connect_login', locals: { url: france_connect_particulier_path }
 
-    = f.label :email, "Email (nom@site.com)"
+    = f.label :email, t('views.sessions.new.email')
     = f.text_field :email, type: :email, autocomplete: 'username', autofocus: true
 
-    = f.label :password, "Mot de passe (#{PASSWORD_MIN_LENGTH} caractères minimum)"
+    = f.label :password, t('views.sessions.new.password', min_length: PASSWORD_MIN_LENGTH)
     = f.password_field :password, autocomplete: 'current-password'
 
     .auth-options
       %div
         = f.check_box :remember_me
-        = f.label :remember_me, "Se souvenir de moi", class: 'remember-me'
+        = f.label :remember_me, t('views.sessions.new.remember_me'), class: 'remember-me'
 
       .text-right
-        = link_to "Mot de passe oublié ?", new_user_password_path, class: "link"
+        = link_to t('views.sessions.new.reset_password'), new_user_password_path, class: "link"
 
-    = f.submit "Se connecter", class: "button large primary expand"
+    = f.submit t('views.sessions.new.connexion'), class: "button large primary expand"
 
   %hr
   %p.center
-    %span Vous êtes nouveau sur #{APPLICATION_NAME.gsub("-","&#8209;").html_safe} ?
+    %span= t('views.sessions.new.message_procedure', app_name: APPLICATION_NAME.gsub("-","&#8209;")).html_safe
```

Comment:
Pinaillage : y'a pas de règle stricte sur le nommage des clefs de locales. Mais généralement c'est une bonne idée que le nom de la clef corresponde à un résumé très bref du message.

Ici je proposerais bien de nommer cette clef `views.sessions.new.are_you_new`, ou quelque chose comme ça.

Reply:
C'est noté :)
Aux lignes 50-52 des fichiers .yml, j'ai mis en clé ligne1, ligne2, ligne3.
Est-ce qu'il faut également les rendre plus explicites ?

Reply:
Pour ces ligne1, ligne2, ligne3 quelqu'un se souvient pourquoi on fait du word-wrap forcé ? On ne peut pas s'en sortir avec des non-breaking spaces au pire ?

Reply:
Je pense qu'on pourrait faire du NBSP (ou même mettre les <br> dans la locale), mais je dirais que c'est hors-scope de cette PR.

Reply:
ok